src/Zynga/Framework/Type/V1/UInt64Box.hh

```diff
@@ -21,7 +21,7 @@ class UInt64Box extends BaseBox {
     $this->setIsDefaultValue(true);
 
     if ($value !== null) {
-      $this->importFromInteger($value);
+      $this->set($value);
```

Comment:
This change breaks this box so that it no longer has a valid default value flag. From reading your commit, I don't think you meant to do this? Did you mean to no longer have default Uint64box values?

Reply:
Patrick and I spoke about this. He brought up that in the current case for UInt64 box, it considers 0 to be the default value. Hence if we feed the constructor any value, it is implictly no longer a default.

If we wanted to change the box so that we can set it's default value and state, we would have to inherit from this class and modify the reset function to do so. I would suggest we make an API change in the future so that we can restore state without needing inheritance.